markdown/chapters/meandeck/simple-piles.md

```diff
@@ -0,0 +1,340 @@
+you have the available mana it may help you against opponents who are either
+trying to slow roll their permission spells or who are holding something like
+:card[Stifle].
+
+Any pile that uses :card[LED] has an inherent weakness where you are unable to
+hold anything in hand and these are generally for when you know they don't have
+any viable interaction or you are unable to take things slower.
+
+We also get to be introduced to one of the more powerful cantrips of the deck;
+:card[Consider].
+
+## Deep Analysis Piles
+
+> {U} + Consider
+
+:::row{variant=PILE}
+DA
+LED
+SW
+LP
+TO
+:::
+
+If you are low on resources you can utilise the above pile using :card[Consider]
+in conjunction with :card[Deep Analysis]. Consider allows you to put Deep
+Analysis into your graveyard and draw the Lion's Eye Diamond. You can then cast
+the Deep Analysis with Flashback in order to draw into additional mana and the
+Oracle. You can of course replace Petal with Cavern of Souls if you have a land
+drop available.
+
+Let's walk through it:
+
+1. Cast Consider, surveil Deep Analysis, draw LED
+1. Cast LED, crack LED for {U}{U}{U}
+1. Cast Deep Analysis for its Flashback Cost target yourself, draw Street Wraith
+   and Petal
+1. Cast Lotus Petal, crack for {U}
+1. Cycle Street Wraith, draw Thassa's Oracle
+1. Cast Thassa's Oracle and win the game with 0 cards left in your deck
+
+Similarly, you can replace the Consider in hand with any cantrip effect.
+
+> {U} + Cantrip
+
+:::row{variant=PILE}
+Consider
+DA
+LED
+LP
+TO
+:::
+
+1. Cantrip to draw into your pile, draw Consider
+1. Cast Consider, surveil Deep Analysis, draw LED
+1. Cast LED, crack LED for {U}{U}{U}
+1. Cast Deep Analysis for its Flashback Cost target yourself, Petal and Oracle
+1. Cast Lotus Petal, crack for {U}
+1. Cast Thassa's Oracle and win the game with 0 cards left in your deck
+
+You can also interchange other requirements depending on what you have in hand.
+If you have :card[Deep Analysis] in hand for example you can cantrip into the
+LED first and just treat it like any other draw 2 pile.
+
+## Generic Cantrip Piles
+
+There are some circumstances where maybe you have decided the mutliple draw
+spells are not good enough to include in your decklist or maybe you have pitched
+your preferred one to a :card[FoW]. There are still plenty of options you can
+explore in making piles without them.
+
+Let's look at one of the simplest.
+
+> {U}{U} + Cantrip
+
+:::row{variant=PILE}
+SW
+SW
+EoA
+EoA
+TO
+:::
+
+This is a basic as you can get and is just as mana efficient as the first
+pile presented in this chapter. It does have the same weakness in that it
+doesn't allow you to play around any interaction but it can have its useful
+moments. Use sparingly.
+
+1. Cantrip to draw into your pile, draw Street Wraith
+1. Cycle Street Wraith, draw Street Wraith
+1. Cycle Street Wraith, draw Edge of Autumn
+1. Cycle Edge of Autumn, draw Edge of Autumn
+1. Cycle Edge of Autumn, draw Thassa's Oracle
```

Comment:
```suggestion
1. Cycle 4 times and draw Thassa's Oracle
```

Reply:
I think I want to keep it verbose here, really spell it out. In my mind it helps people think about the cost if they are working through of each 2 life payment or land sacrifice involved.